.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+</details>
+
+<br>
+
+<details>
+<summary>Ways to contribute :mag:</summary>
+<br>
+
+To contribute to the Python client library on GitHub, you can:
+
+- Report a bug
+- Suggest a new feature
+- Build a new feature
+- Contribute to the documentation
+</details>
+
+<br>
+
+<details>
+<summary>Report a bug :bug:</summary>
+<br>
+
+This section guides you through reporting a bug. Following these guidelines helps maintainers and the community understand the bug, reproduce the behavior, and find related bugs.
+
+### Before reporting a bug
+
+Please check the following list:
+
+- **Do not open a GitHub issue for [security vulnerabilities](SECURITY.MD)**, instead, please contact us at [security@iota.org](mailto:security@iota.org).
+
+- **Ensure the bug was not already reported** by searching on GitHub under [**Issues**](https://github.com/iotaledger/iota.py/issues). If the bug has already been reported **and the issue is still open**, add a comment to the existing issue instead of opening a new one. You can also find related issues by their [label](https://github.com/iotaledger/iota.py/labels?page=1&sort=name-asc).
+
+**Note:** If you find a **Closed** issue that seems similar to what you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Submitting A Bug Report
+
+To report a bug, [open a new issue](https://github.com/iotaledger/iota.py/issues/new), and be sure to include as many details as possible, using the template.
+
+**Note:** Minor changes such as fixing a typo can but do not need an open issue.
+
+If you also want to fix the bug, submit a [pull request](#pull-requests) and reference the issue.
+</details>
+
+<br>
+
+<details>
+<summary>Suggest a new feature :bulb:</summary>
+<br>
+
+This section guides you through suggesting a new feature. Following these guidelines helps maintainers and the community collaborate to find the best possible way forward with your suggestion.
+
+### Before suggesting a new feature
+
+**Ensure the feature has not already been suggested** by searching on GitHub under [**Issues**](https://github.com/iotaledger/iota.py/issues).
+
+### Suggesting a new feature
+
+To suggest a new feature, talk to the IOTA community and IOTA Foundation members on [Discord](https://discord.iota.org/).
+
+If the team members approves your feature, they will create an issue for it.
+</details>
+
+<br>
+
+<details>
+<summary>Build a new feature :hammer:</summary>
+<br>
+
+This section guides you through building a new feature. Following these guidelines helps give your feature the best chance of being approved and merged.
+
+### Before building a new feature
+
+Make sure to discuss the feature with the developers on [Discord](https://discord.iota.org/).
+
+Otherwise, your feature may not be approved at all.
+
+### Building a new feature
+
+To build a new feature, check out a new branch based on the `master` branch, and be sure to consider the following:
```

Comment:
```suggestion
To build a new feature, check out a new branch based on the `develop` branch, and be sure to consider the following:
```

❔ So that the new feature takes into account any changes made since the previous release?